.github/workflows/devnet-deploys.yml

```diff
@@ -234,6 +241,41 @@ jobs:
           docker tag aztecprotocol/aztec:${{ env.DEPLOY_TAG }} aztecprotocol/aztec:${{ github.sha }}
           docker push aztecprotocol/aztec:${{ github.sha }}
 
+  build-aztec-arm:
+    needs: [set-network, setup-arm]
+    env:
+      BRANCH_NAME: ${{ needs.set-network.outputs.branch_name }}
+      DEPLOY_TAG: ${{ needs.set-network.outputs.deploy_tag }}
+      TF_VAR_DEPLOY_TAG: ${{ needs.set-network.outputs.deploy_tag }}
+      API_KEY: ${{ secrets[needs.set-network.outputs.network_api_key] }}
+      TF_VAR_API_KEY: ${{ secrets[needs.set-network.outputs.network_api_key] }}
+      API_KEY_NAME: ${{ needs.set-network.outputs.network_api_key }}
+    runs-on: ${{ github.actor }}-arm
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: "${{ env.GIT_COMMIT }}"
+          fetch-depth: 0
+      - uses: ./.github/ci-setup-action
+        with:
+          concurrency_key: build-aztec-${{ github.actor }}
+          dockerhub_password: "${{ env.DOCKERHUB_PASSWORD }}"
+      - name: "Build & Push aztec images"
+        timeout-minutes: 40
+        if: ${{ github.event.inputs.no_rebuild_images == 'false' }}
+        run: |
+          env
+          earthly-ci --no-output --push ./yarn-project+export-aztec-arch --DIST_TAG=${{ env.DEPLOY_TAG }} --ARCH=linux/arm64
```

Comment:
I don't think it's as simple as this. We will need to publish all images with the appropriate arch and then build and publish a manifest. The manifest means when you do `docker pull aztecprotocol/aztec:devnet` it automagically gives you the image for your architecture. 

We do this in CCI henc why it works for the sandbox etc.

Reply:
Hmm, looking at the Earthly docs, it's possible that it is already reating a manifest as we don't specify `--no-manifest-list`.  So maybe we do just need to specify the arch, but presumably we need to do it on all builds.

Reply:
Exactly, I was under the impression Earthly creates a manifest all the time